modules/RTC/RTCUtils.js

```diff
@@ -459,13 +490,21 @@ function newGetConstraints(um = [], options = {}) {
             constraints.video = {};
         }
 
+        const {
+            frameRate = {
+                min: SS_DEFAULT_FRAME_RATE,
+                max: SS_DEFAULT_FRAME_RATE
+            }
```

Comment:
What happens in the case where only max or min is not defined, but the frameRate option is defined?

Reply:
We won't set the undefined value. Do you think we should handle this? It is getting pretty complicated if we want to handle it. For example if the minimum is not set but the maximum is set to 3, we can't use the default value for the minimum because it is 5. I'm not sure adding such a complexity in the code is worth it and maybe the user don't want to set any constraint about the undefined value.

Reply:
I don't care much about handling it now but I'm wondering what the side effects are, if any.

Reply:
Oh. OK. Sorry I didn't understood from the first time :) I'll test to check.